aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -459,7 +459,7 @@ std::vector<Dimname> broadcast_to_outnames(
   return unify_from_right(reference_names, tensor_names);
 }
 
-std::vector<Dimname> compute_cat_outnames(ITensorListRef tensors) {
+std::vector<Dimname> compute_cat_outnames(ITensorList tensors) {
```

Comment:
Hmm, this doesn't look like it was supposed to happen...

Reply:
Ah, I thought it would be easier to keep the old names in these PRs. Clearly, I was wrong.